src/main/java/com/zzpj/backend/entities/Alcohol.java

```diff
@@ -0,0 +1,52 @@
+package com.zzpj.backend.entities;
+
+import javax.persistence.*;
+import javax.validation.constraints.Min;
+import java.util.UUID;
+
+@Entity
+public class Alcohol {
+
+    @Id
+    private UUID id;
+    @Column(unique = true)
+    private String name;
+    @Column
+    @Min(0)
+    private double cost;
+    @Column
+    @Min(0)
+    private int amount;
+
+    public UUID getId() {
```

Comment:
Definitely we should use [Lombok ](https://projectlombok.org/features/GetterSetter) replace getters and setters